ipalib/util.py

```diff
@@ -1203,17 +1203,32 @@ def get_terminal_height(fd=1):
         return os.environ.get("LINES", 25)
 
 
-def open_in_pager(data):
+def get_pager():
+    """ Get path to a pager
+
+    :return: path to the file if it exists otherwise None
+    :rtype: str
+    """
+
```

Comment:
Style nit pick: Please remove the extra whitespace between doc string and first code.